cloudbuild.yaml

```diff
@@ -0,0 +1,5 @@
+steps:
+- name: 'gcr.io/cloud-builders/docker'
+  entrypoint: bash
+  dir: build
+  args: ['-ex', 'build_docker.sh']
```

Comment:
This should set `NO_PODMAN_PULL=1`.

Looking at the logs indicates that Cloud Build doesn't support running `podman` (at least not in its current configuration).